### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]:NVDA and Jaws are  not announcing the content present in  create Service Principle status Window sequentially

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1512]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Tab to Tools menu, point to "Azure", and then select "Azure Sign In".
4. In the Azure Sign In window, select "Automated" radio button, and navigate to "New" button and select it.
5.  In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. After sign In Create Authentication Files window will opened .
7. In "Create Authentication files window" navigate and select the "subscriptions" that you want to use, choose your "destination directory", and then select Start.
8. In the Create Service Principle status," Press "Insert+B" to read  the entire window then observe the NVDA and Jaws Behavior

**Expected Result:**
NVDA and JAWS Should announce in the following manner

- Create Service Principal Status  dialog  
- table
  Step header
  Result header
 Details header
 tenant ID: 72f988bf-86f1-41af-91ab-2d7cd011db47 ===
 Creating Azure Active Directory application
 SUCCESSFUL
 appId: c2adb1fe-f9d9-42b2-b33c-884027cbdcb6; appObjectId: 7060bf84-d663-464e-b2f6-   b7e1a1fb2a22
 Creating service principal for application
 SUCCESSFUL
 spObjectId: a7fd54b0-7ee0-4776-a486-c7020ccabf09
 Role assignment for subscription bd62906c-0a81-43c3-a2f8-126e4cf66ada
 FAILED
 (AuthorizationFailed) The client 'v-gudars@microsoft.com' with object id '7f30850a-6188-48c5-882b-594271eedac9' does not have authorization to perform action 'Microsoft.Authorization/roleAssignments/write' over scope '/subscriptions/bd62906c-0a81-43c3-a2f8-126e4cf66ada'.
Role assignment for subscription 7f31cba8-b597-4129-b158-8f21a7395bd0
SUCCESSFUL
Assigned as Contributor
Overall status
FAILED
Can't create a service principal.
- scroll bar  
- Created authentication file(s) (one per Active Directory instance/tenant):
- list
  list Items 
- OK  button
- Cancel  button

**Actual Result:**
**NVDA Behavior:**
NVDA is announcing as
Create Service Principal Status  dialog  Created authentication file(s) (one per Active Directory instance/tenant):
table
tenant ID: 72f988bf-86f1-41af-91ab-2d7cd011db47 ===
Creating Azure Active Directory application
SUCCESSFUL
appId: c2adb1fe-f9d9-42b2-b33c-884027cbdcb6; appObjectId: 7060bf84-d663-464e-b2f6-b7e1a1fb2a22
Creating service principal for application
SUCCESSFUL
spObjectId: a7fd54b0-7ee0-4776-a486-c7020ccabf09
Role assignment for subscription bd62906c-0a81-43c3-a2f8-126e4cf66ada
FAILED
(AuthorizationFailed) The client 'v-gudars@microsoft.com' with object id '7f30850a-6188-48c5-882b-594271eedac9' does not have authorization to perform action 'Microsoft.Authorization/roleAssignments/write' over scope '/subscriptions/bd62906c-0a81-43c3-a2f8-126e4cf66ada'.
Role assignment for subscription 7f31cba8-b597-4129-b158-8f21a7395bd0
SUCCESSFUL
Assigned as Contributor
=== Overall status
FAILED
Can't create a service principal.
scroll bar  0
Step
Result
Details
Created authentication file(s) (one per Active Directory instance/tenant):
list
filler
OK  button
Cancel  button
**Jaws Behavior:**
Create Service Principal Status dialog
Step Result Details Created authentication file(s) (one per Active Directory instance/tenant): 
Cancel Button

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![hierarchy is incorrect for create service principle status dialog](https://user-images.githubusercontent.com/22956130/36149888-82bb33d4-10e7-11e8-9847-0d86f7c82123.jpg)
![create service principle status](https://user-images.githubusercontent.com/22956130/36149893-8ba6df0c-10e7-11e8-830c-63ab92b808ed.jpg)

**Comment by Eskibear** [comment]
under investigation

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749983
